crypto/fipsmodule/bn/bn_test.cc

```diff
@@ -2792,8 +2792,18 @@ TEST_F(BNTest, RSAZABI) {
     return;
   }
 
-  alignas(64) BN_ULONG table[32 * 18] = {0};
-  alignas(64) BN_ULONG rsaz1[40], rsaz2[40], rsaz3[40], n_rsaz[40];
+  stack_align_type buffer_table[64 + (sizeof(BN_ULONG) * (32*18))] = {0};
```

Comment:
Isn't adding 63 bytes sufficient for the alignment?

Reply:
technically, yes. And practically, for that matter. When I wrote this, I guess I just want to make it obvious, this was the desired alignment. But you could drop it be 1.